Storage/src/Bucket.php

```diff
@@ -242,7 +254,10 @@ public function exists()
      *           at the [JSON API docs](https://cloud.google.com/storage/docs/json_api/v1/objects/insert#request-body).
      *     @type array $metadata['metadata'] User-provided metadata, in key/value pairs.
      *     @type string $encryptionKey A base64 encoded AES-256 customer-supplied
-     *           encryption key.
+     *           encryption key. If you would prefer to manage encryption
+     *           utilizing the Cloud Key Management Service (KMS) please use the
+     *           $metadata['kmsKeyName'] setting. Please note if using KMS the
+     *           key ring should use the same location as the bucket.
```

Comment:
Small documentation nit: @dwsupplee, please update the word "should" to "must".

Reply:
All set!

Reply:
Thanks, LGTM